library/md5.c

```diff
@@ -361,6 +393,13 @@ int mbedtls_md5_ret( const unsigned char *input,
     return( ret );
 }
 
+void mbedtls_md5( const unsigned char *input,
```

Comment:
This needs to be guarded by `! MBEDTLS_DEPRECATED_REMOVED`, too.

Reply:
Fixed